content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -7,6 +7,8 @@ weight = 60
 
 ## Introduction
 
+Kubeflow is built-in on MicroK8s.
```

Comment:
Sorry, KF can exist without MicroK8s and the original first paragraph (below) already covers that MicroK8s can be used as one of the ways to deploy KF locally. So this sentence may not be necessary (and I'm no 100% sure about the correct grammatical use of "built-in" here, though I may be wrong)

Reply:
Hey @8bitmp3, the idea here is that besides being an underlying K8s, as for instance MiniKube, MicroK8s comes with Kubeflow already built-in as an add-on. That is what allows for `microk8s enable kubeflow`

Reply:
WDYT?

Reply:
> MicroK8s comes with Kubeflow already built-in as an add-on.

Gotcha. I'm with you. We should probably use what you mentioned and rephrase it, since it's so 🚀 Let's add this sentence to the introductory sentence (in the same first paragraph) like so:

```diff
This guide describes how to deploy and run Kubeflow using [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster — locally on built-in hypervisors.
+ MicroK8s comes with Kubeflow already built-in as an add-on.
```